dd-sdk-android/src/main/kotlin/com/datadog/android/rum/webview/RumWebChromeClient.kt

```diff
@@ -19,6 +19,19 @@ import com.datadog.android.rum.RumErrorSource
  *
  * Any console message will be forwarded to an internal [Logger], and errors
  * will be sent to the [GlobalRum] monitor as RUM Errors.
+ *
+ * This class is deprecated and you should use the new
+ * [com.datadog.android.webview.DatadogEventBridge] to track your WebViews:
+ *
+ * ```kotlin
+ * val configuration = Configuration.Builder().setWebViewTrackingHosts(listOf(<YOUR_HOSTS>))
+ * Datadog.initialize(this, credentials, configuration, trackingConsent)
+ *
+ * webView.webViewClient = WebViewClient()
+ * webView.webChromeClient = WebChromeClient()
```

Comment:
What is the point of doing that? The default `WebViewClient` and `WebChromeClient` are no-op classes, so why should we advise people to set them here?